### DISCUSSION TITLE
REopt Web Tool Unexpected Error Message

### DISCUSSION THREAD

**Discussion by cp1119** [NREL/REopt-API-Analysis#72] [1 - General REopt Questions] [answered]
Hi, even when running the sample site and only using the default values, I receive the attached error when trying to get the results. 
<img width="901" alt="Screenshot 2022-08-04 222453" src="https://user-images.githubusercontent.com/92208509/182945916-90b09709-cd7b-42c3-aa50-01f54c846989.png">

I tried simulating for the following outputs: 
![image](https://user-images.githubusercontent.com/92208509/182946889-cf18ef3c-58c9-4d65-8053-cc43bf1e4f90.png)

I am not sure what I am doing incorrectly/what might be wrong about my inputs to run the web tool. Thank you for any help! :)

**Comment by adfarth** [comment]
Hello, we are experiencing server-related errors that are causing the error you're seeing on the REopt web tool. We apologize for this inconvenience and are working to get the web tool back up and running.

**Reply by cp1119** [reply]
Thank you for the quick answer! Please post when the tool is back up and running. Thank you.

**Comment by adfarth** [comment, marked as answer]
It should now be back up and running.

**Reply by cp1119** [reply]
The tool works again for me. Thank you!
